src/app/src/main/kotlin/org/vaccineimpact/api/app/OneTimeLink.kt

```diff
@@ -45,6 +45,7 @@ data class OneTimeLink(val action: OneTimeAction,
                     OneTimeAction.COVERAGE -> stream(controllers.modellingGroup.getCoverageData(context, repos.modellingGroup), context)
                     OneTimeAction.DEMOGRAPHY -> stream(controllers.touchstone.getDemographicData(context, repos.touchstone), context)
                     OneTimeAction.SET_PASSWORD -> controllers.password.setPasswordForUser(context, repos.user, context.params("username"))
+                    OneTimeAction.MODEl_RUN_PARAMETERS -> throw UnsupportedOperationException(OneTimeAction.MODEl_RUN_PARAMETERS.toString())
```

Comment:
I updated the models submodule to get the `AllColumnsRequired` annotation required by `DataTableDeserializer`, and had to add this case to satisfy exhaustiveness in the when expression. It can stay there until i940 is finished.